docs/user/metrics/features.csv

```diff
@@ -8,3 +8,4 @@ Library,ML Backend,I/O Backend,Spatial Backend,Transform Backend,Datasets,Weight
 `TerraTorch`_,PyTorch,"GDAL, h5py, pandas, xarray",R-tree,Albumentations,22,1,✅,✅,❌,🚧
 `SITS`_,R Torch,GDAL,-,tidyverse,22,0,❌,✅,✅,✅
 `scikit-eo`_,"scikit-learn, TensorFlow","pandas, scipy, numpy, rasterio","geopandas",numpy,0,0,❌,❌,❌,🚧
+`GeoAI`_,"PyTorch","GDAL, rasterio, xarray","geopandas, STAC",Albumentations,0,6,❌,✅,✅,❌
```

Comment:
* **I/O Backend**: I usually combine GDAL and rasterio into GDAL since rasterio is just a wrapper around GDAL. 
* **Spatial Backend**: Are both geopandas and STAC used for spatiotemporal indexing? Or is one of them used for file I/O?

Reply:
Good points. Revised accordingly.